### PR TITLE
Add PBS Affiliates

### PR DESCRIPTION
**Pull request by MapGuy11**
More to Come!

### CODE REVIEW

**Review by MapGuy11 — COMMENTED** [review]
#5078792 is actually Alabama

File: channels/us.m3u [hunk 1]
```diff
@@ -3697,3 +3697,77 @@ https://rockentertainment-zoomoo-1.samsung.wurl.com/manifest/playlist.m3u8
...
+https://wfsuhd.lls.cdn.pbs.org/wfsuhd/da7db8ae-5c3b-450e-9a25-76aa661eac79/primary.m3u8
+#EXTINF:-1 tvg-id="WEDU.us" tvg-name="WEDU-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wedu-color-cobranded-logo-87oN0he.png" group-title="Public Broadcasting",PBS Tampa (WEDU) (Geo)
+https://wedudt.lls.cdn.pbs.org/wedudt/c4ab80c1-ca8c-4a6e-a8b5-c5995919792b/primary.m3u8
+#EXTINF:-1 tvg-id="WVAN.us" tvg-name="WVAN-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wgtv-color-single-brand-logo-IeTkPR9.png" group-title="Public Broadcasting",PBS Georgia (WVAN) (WXGA) (Geo)
+https://wgtvdt.lls.cdn.pbs.org/wgtvdt/a4907776-c769-452f-ab2f-74bb1f86676f/primary.m3u8
+#EXTINF:-1 tvg-id="WBPA.us" tvg-name="WPBA-DT1" tvg-country="US" tvg-language="English" tvg-logo="https://image.pbs.org/stations/wpba-color-single-brand-logo-d751apM.png" group-title="Public Broadcasting",PBS Atlanta (WBPA) (Geo) (National PBS Feed)
+https://pbs.lls.cdn.pbs.org/est/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/us.m3u:3771]: 3 different call signs but same link, is this a multicast one ?

Comment (MapGuy11) [reply channels/us.m3u:3771]: No there really placeholders as PBS I think is requiring streams in the near future for all affiliates.

**Comment by LaneSh4d0w** [comment]
The "(Geo)" label you've wrote should be actually "[Geo-blocked]"

**Comment by MapGuy11** [comment]
> [Geo-blocked]

Thanks!

**Comment by freearhey** [comment]
@MapGuy11 Is this PR not ready yet?

**Comment by MapGuy11** [comment]
> @MapGuy11 Is this PR not ready yet?

I can add it but it doesn't have all 50 states.

**Comment by freearhey** [comment]
@MapGuy11 So you plan to add channels for all states at one time? If so, we can keep PR open as long as necessary. I just didn't get that from the description.

**Comment by MapGuy11** [comment]
> @MapGuy11 So you plan to add channels for all states at one time? If so, we can keep PR open as long as necessary. I just didn't get that from the description.

Yeah sorry.

**Comment by LaneSh4d0w** [comment]
Merging conflict occurs in this PR.

Should I merge the already added channels when I'm done fixing ?

**Comment by MapGuy11** [comment]
> Merging conflict occurs in this PR.
> 
> Should I merge the already added channels when I'm done fixing ?

Sure.

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**